.github/pull_request_template.md

```diff
@@ -3,3 +3,16 @@
 ## What are you trying to address
 
 - Write a detailed description of all changes - and if appropriate, why they are needed
+
+
+## Checklist
+
+[READY TO PR? Use the check-list below to ensure your branch is ready for PR.]
+
+- [ ] Changes follow the repo structure and land in the appropriate folder and section
+- [ ] No lint check errors
+- [ ] No confidential information
+- [ ] No duplicated content
+- [ ] Labeled appropriately
+- [ ] Added 2 reviewers
+- [ ] Not a developer blog
```

Comment:
The section on "not a developer blog" is a bit out of place in the checklist, I think we can just validate that one manually... that might raise more questions than it fixes. I would suggest removing it... I can bring this up with the authors on an as needed bases instead, since it has only happened once

Reply:
I would also maybe suggest, if we are adding content to this (we removed it before because it was getting so long) - that we mention that it is normal for the link checker to fail on links not related to your content... don't worry about this, the reviewer will help you resolve it

Reply:
Removed "not a developer blog" as suggested and added a note on link checker.

Reply:
> "not a developer blog"

I'm curious now 😅, did this come from some PR descriptions sometimes becoming "long stories" or personal opinion being overly expressed?

P.S. love this repo, it's such a valuable resource. 🙌

Reply:
> > "not a developer blog"
> 
> I'm curious now 😅, did this come from some PRs becoming "long stories" or personal opinion being overly expressed?
> 
> P.S. love this repo, it's such a valuable resource. 🙌

No, this came from an internal user story. There have been some cases in the past where people have been confused about whether they should share to this repo or our developer blog. The issue was that people shared content that fit better here to the blog, but the reverse hasn't been an issue.

Thank you for the feedback on the repo btw :) much appreciated

Reply:
> Removed "not a developer blog" as suggested and added a note on link checker.

I saw you changed it to "No lint check errors related to your changes". This is not really what I meant.
Suggestion:

No lint errors
No link check errors related to your changes.  

> Note: you may see link check errors on pages you have not touched. This is normal, and due to either broken links or sites that reject link checker bots. The reviewer will help you get to a green state on these.

Reply:
I had the note below the check list. 

I updated the check list so the "No link check errors" are now at the bottom and next to the note (which I also reworded to your suggestion). 

Let me know if that works!